.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request
+about: Suggest an enhancement to the ohsome API.
+title: ''
+labels: 'feature request'
```

Comment:
currently, we don't have such a [label](https://github.com/GIScience/ohsome-api/labels). I think `enhancement` would be used?!

```suggestion
labels: 'enhancement'
```

Reply:
`enhancement` is also ok since the description of the `enhancement` label says `New feature or request`

Reply:
> AFAIK, only collaborators can edit the labels of an issue (except for the pre-assigned basic "issue type" label from the respective template, e.g., `bug`). Therefore, I would suggest to drop the sections requesting the addition of labels.

Having the possibility to set the priority was a request by @SlowMo24, since we use labels I thought just to draw attention to the use of the labels. No problem about deleting this part. @SlowMo24 what do you think?

Reply:
Same for the bug report

Reply:
I'm fine with anything but having some sort of 'how urgent' information given by the user **may** be of value. Just to see the difference between 'I need this for my research tomorrow' and 'That would also be great, but I wont use it til next year and there is a workaround'. It should though be clear that we have our own prioritisation and obey noone ;-)